### PR TITLE
Add pre-commit configuration.

### PR DESCRIPTION
**Pull request by michal-samluk**
Allow setup project locally for pre-commit.

Rerganise the help box for better reusability (can be used in GH and
CLI).

TODO: 
* [x] pre-commit project setup 
* [x] pre-commit local setup

### CODE REVIEW

**Comment by andrzejtrzaska** [comment]
:

**Comment by kruszczynski** [comment]
:white_check_mark:

File: lib/codeguard.rb [hunk 1]
```diff
@@ -1,14 +1,18 @@
...
 require 'codeguard/js_hint'
 require 'codeguard/rubocop'
 require 'codeguard/scss_lint'
 require 'codeguard/git_message'
 require 'codeguard/diff'
+require 'codeguard/pre_commit'
 
 module Codeguard
-  LINTERS = [Coffeelint, GitMessage, JSHint, Rubocop, SCSSLint]
+  LINTERS = [Coffeelint, GitMessage, JSHint, PreCommit, Rubocop, SCSSLint]
```
Comment (pjar) [inline lib/codeguard.rb]: Maybe don't duplicate GitMessage and PreCommit

Comment (michal-samluk) [inline lib/codeguard.rb]: hmmm why?

**Pull request merged**